doc/dev/tests.md

```diff
@@ -5,24 +5,42 @@ title: Tests
 
 ## Overview
 
-Npgsql comes with an extensive test suite to make sure no regressions occur. All tests are run on our build server on all supported .NET versions (including a recent version of mono) and all supported PostgreSQL backends.
+Npgsql has an extensive test suite to guard against regressions. The test suite is run on the official build server for all supported .NET versions (including a recent version of mono) and all supported PostgreSQL backends.
 
-There is also a growing suite of speed tests to be able to measure performance. These tests are currently marked [Explicit] and aren't executed automatically.
+Npgsql also includes a growing suite of performance tests. These tests are marked `[Explicit]` and can be executed manually.
```

Comment:
Uh, this is old and should be killed off. Performance tests are no longer implemented as unit tests, but rather with BenchmarkDotNet (Npgsql.Benchmarks).